SentryTestUtils/MainThreadTestIntegration.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import Sentry
+
+public class MainThreadTestIntegration: NSObject, SentryIntegrationProtocol {
+
+    public var installedInTheMainThread = false
+
+    public func install(with options: Options) -> Bool {
+        installedInTheMainThread = Thread.isMainThread
+        return true
+    }
+
+    public static func replaceOptionIntegrations(_ options: Options) {
```

Comment:
I don't understand what is meant by "replace option"

Reply:
it replaces all integrations in an Option object.
Suggestions are welcome.

Reply:
```suggestion
    public static func replaceIntegrationsOn(_ options: Options) {
```